### PR TITLE
Update ai.json

### PR DESCRIPTION
**Pull request by talves**
Looking at some of the AGI terms I've read about.

### CODE REVIEW

File: ipsums/ai.json [hunk 1]
```diff
@@ -132,5 +132,23 @@
...
+	"AGI",
+	"strong AI",
+	"full AI",
+	"general intelligence action",
+	"sentience",
+	"consciousness",
+	"artificial consciousness",
+	"weak AI",
+	"general cognitive abilities",
+	"transformitive AI",
```
Comment (cassidoo) [inline ipsums/ai.json]:
```suggestion
	"transformative AI",
```

File: ipsums/ai.json [hunk 2]
```diff
@@ -132,5 +132,23 @@
...
+	"turing test",
+	"cognitive science",
+	"AI complete",
+	"AI hard",
+	"singularity",
+	"AI alignment",
+	"AI safety"
```
Comment (cassidoo) [inline ipsums/ai.json]:
```suggestion

```

this is a repeat

**Review by cassidoo — CHANGES_REQUESTED** [review]
Minor comments! Thanks for your PR!

File: ipsums/ai.json (hunk 1 above)
Comment (talves) [reply ipsums/ai.json]: oops

File: ipsums/ai.json (hunk 2 above)
Comment (talves) [reply ipsums/ai.json]: Missed it. Good catch 😀

**Review by cassidoo — APPROVED** [review]
Heck yeah, thank you!

**Pull request merged**